### PR TITLE
Update confirmation page layout for blockaid alerts

### PR DESCRIPTION
**Pull request by jpuri**
Update confirmation pages layout for blockaid alerts:

https://github.com/MetaMask/metamask-mobile/assets/2182307/1fc2414c-8fbc-43a9-be08-c7734732d7f1

https://github.com/MetaMask/metamask-mobile/assets/2182307/07c01a8e-3316-4d0b-8639-c155c885366b

### CODE REVIEW

File: app/components/UI/TransactionReview/index.js [hunk 1]
```diff
@@ -85,10 +85,10 @@ const createStyles = (colors) =>
       ...fontStyles.bold,
     },
     actionViewWrapper: {
-      height: Device.isMediumDevice() ? 170 : 290,
+      height: Device.isMediumDevice() ? 470 : 550,
     },
     actionViewChildren: {
-      height: 220,
+      height: Device.isMediumDevice() ? 390 : 470,
```
Comment (blackdevelopa) [inline app/components/UI/TransactionReview/index.js:91]: We might need to test these styles across different device sizes. Other than that, looks good to me

**Review by blackdevelopa — DISMISSED** [review]

**Comment by blackdevelopa** [comment]
Hi @jpuri, just noticing that for token approval, the style is a bit off
Screenshot 2023-09-08 at 15 08 52

File: app/components/UI/TransactionReview/index.js (hunk 1 above)
Comment (jpuri) [reply app/components/UI/TransactionReview/index.js:91]: I can test on simulators that I have.

**Comment by jpuri** [comment]
> Hi @jpuri, just noticing that for token approval, the style is a bit off Screenshot 2023-09-08 at 15 08 52

Yeh that is an existing thing @blackdevelopa

**Comment by seaona** [comment]
@blackdevelopa this is an existing issue and we have a ticket here for that https://github.com/MetaMask/metamask-mobile/issues/7178

File: app/components/UI/TransactionReview/index.js (hunk 1 above)
Comment (jpuri) [reply app/components/UI/TransactionReview/index.js:91]: Hey @blackdevelopa : I looked more into this, small devices have `height < 736`, I could not find a simulator for that, but I expect the height configurations here to work fairly well in small height.

@blackdevelopa , @seaona : do you guys have suggestions for small mobile devices using which we can test.

**Review by blackdevelopa — APPROVED** [review]

**Review by segun — APPROVED** [review]

**Pull request merged**